### DISCUSSION TITLE
Item Identification: separate toggle for finished and unf potions

### DISCUSSION THREAD

**Discussion by janne-kemppainen** [runelite/runelite#16825] [Ideas]
I'd like to see names of unf potions but I dont need it for finished potions.

**Comment by ArtificiallyThrown** [comment]
I was just coming here to suggest this, glad but also upset I found a post over a year old on the topic, such a shame, I'd really like this feature!

**Comment by Nightfirecat** [comment]
I've promoted this to an enhancement ticket here: #18204
